site/content/gitops-quickstart/01-apply-gitops.md

```diff
@@ -0,0 +1,258 @@
+
+### Configuring Flux
+
+Here we will install [Flux](https://fluxcd.io), the Kubernetes GitOps
+operator in your cluster. It will take care of deployments for you.
+
+```console
+EKSCTL_EXPERIMENTAL=true eksctl install flux \
+    --git-url <git-url> \
+    --git-email <email-of-committer> \
+    --name <cluster-name>
+```
+
+Additional options to the command are explained in our docs on
+[`install flux`](/usage/experimental/gitops-flux/).
+
+After about a minute your cluster will have `flux` running, which will
+monitor your git repository once you added a deploy key to e.g. Github.
+
+This key can be found at the end of the output of the command, this
+might for example be:
+
+```console
+$ EKSCTL_EXPERIMENTAL=true eksctl install flux \
+    --git-url git@github.com:happy-gopher/flux-get-started \
+    --git-email happy@gopher.org \
+    --name wonderful-wardrobe-1565767990
+[...]
+[ℹ]  Flux will only operate properly once it has write-access to the Git repository
+[ℹ]  please configure git@github.com:happy-gopher/flux-get-started so that the following Flux SSH public key has write access to it
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCt8no0/F3+kD1YukH6sVIv1ONcy9+01G2/AQe1CQA+uRHaioep41U3ghROU7CoM1yTjG+eLYgu26UMvkXVbOmEm+1697adh4qz/yCF0E7JtCIIXGn/1XrLb6OxgtlGKdJ4fTUdxtQSyTvWqjxQhC4ute9hnHWU8oiSrNaq5D20P5x8sgPf4V0A5YWD5S4YliJcIupTzrD7zjhh6TyP5fqhPLHPBZFHStHq0DSD+Gi6vXZz1s9UmuAnxP8pkIlrW22xJyFbsmcjJuks5FvmLo8uJMeWTx5t+3WKWp8ZKrbDJFUWQ8aVMByHYq1c3doevM28CHwz/
+```
+
+Copy the lines starting with `ssh-rsa` and add it as a deploy key, to
+e.g. Github. There you can easily do this in the
+`Settings > Deploy keys > Add deploy key`. Just make sure you check
+`Allow write access` as well.
+
+The next time Flux syncs from git, it will start updating the cluster
+and actively deploying. If you use the [`flux-get-started`
+repo](https://github.com/fluxcd/flux-get-started) from above, here's
+what you will see in your cluster:
+
+```console
+$ kubectl get pods -n demo
+NAME                       READY   STATUS    RESTARTS   AGE
+podinfo-5f4bd464b4-4vf7k   1/1     Running   0          58m
+podinfo-5f4bd464b4-hkgzt   1/1     Running   0          58m
+```
+
+Remember that you can further tweak the installation of Flux as
+discussed in our [`install flux` docs](/usage/experimental/gitops-flux/).
+
+### Handcrafting your configuration
+
+In this second step we will use `eksctl generate config`, so you can
+easily handcraft your cluster configuration and maintain it in Git, just
+like you would for the workloads you are running.
+
+During the previous command (`eksctl install flux`), we instructed Flux
+to watch a config repository. This is where the workloads are defined.
+Now we will add the cluster config as well.
+
+In this example we will run:
+
+```console
+EKSCTL_EXPERIMENTAL=true eksctl generate profile \
+        --name wonderful-wardrobe-1565767990 \
+        --git-url git@github.com:happy-gopher/eks-gitops-example.git \
```

Comment:
To do: we are going to change the name of this repo. Update when we have the new name. @kalbir what new name did you decide?

Reply:
`eks-quickstart-app-dev`

Reply:
Give me a ping when it's moved/renamed.

Reply:
I filed https://github.com/weaveworks/eks-gitops-example/issues/14 for this.

Reply:
https://github.com/weaveworks/aws-workstream/issues/77 it is.